src/components/verticalCard.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Card, Image, Icon } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import moment from 'moment';
+import '../styles/verticalCard.scss';
```

Comment:
Great job Sinmi. How do you feel about importing the SASS file into the index.scss as we all agreed last week? instead of importing it into the component file as you did here?

Reply:
Yes that's true, that's cleaner, I am correcting that right away thanks